src/App.tsx

```diff
@@ -1,109 +1,23 @@
 import React from 'react';
-
+import { Post } from './react-app-env';
+import { PostList } from './components/PostList';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+const posts: Post[] | [] = postsFromServer.map(post => ({
```

Comment:
The type of posts should only be an array of posts .

Reply:
What if the result of Api is null or error? It's a good practice to have an empty array in order to handle the situation...